lxd/instance/drivers/driver_lxc.go

```diff
@@ -2582,6 +2582,11 @@ func (d *lxc) validateStartup(statusCode api.StatusCode) error {
 		return fmt.Errorf("The image used by this instance requires nesting. Please set security.nesting=true on the instance")
 	}
 
+	// Check if instance is start protected.
+	if shared.IsTrue(d.expandedConfig["security.protection.start"]) {
+		return fmt.Errorf("Instance is protected")
```

Comment:
Please can we change this to 

> Instance is protected from being started

Reply:
Sure. And do we need to change [this line](https://github.com/canonical/lxd/blob/5916f950f9afb84a967f9e67b724723d5d8ec6c4/lxd/instance/drivers/driver_qemu.go#L6115) too? Maybe in a different PR?

Reply:
ah yeah, you can do it in the same PR just a different commit please, good spotting.

Reply:
Done